src/lambda_deploy/lambda_deploy.py

```diff
@@ -149,6 +154,10 @@ def deploy(self, *lambdas):
             ),
             os.listdir(self.lambda_dir)
         )
+        
+        if not lambda_dirs and not lambdas:
```

Comment:
So just getting back to this, sorry, busy week last week!  One issue with the approach here is that if you have a Lambda that has subdirectories, this will treat the subdirectories as if *they* were Lambdas, and think you want to package them up.

I suppose the real issue is trying to square how I originally thought of this, as being pointed at a directory *containing* Lambda directories, vs. the idea of it being run *in* a Lambda directory.  I actually think perhaps the way you thought of it is more intuitive - certainly I feel like maintaining both concepts unnecessarily complicates things and leads to corner cases that will cause bugs like the one I identified with this approach.

So I'd say the approach should be to get rid of my fancy little `filter` statement to populate a list of Lambda dirs, and instead package up `os.getcwd` - or rather `self.lambda_dir`, which is set to `os.getcwd` by default, and can be overridden by passing the `-d` option.

Let me know if this sounds good to you.